src/layout/Group/__snapshots__/SummaryGroupComponent.test.tsx.snap

```diff
@@ -12,11 +12,13 @@ exports[`SummaryGroupComponent SummaryGroupComponent -- should match snapshot 1`
       <span
         class="label"
       >
-        Mock group
+        <span>
+          Mock group
+        </span>
       </span>
       <button
         aria-label="Change: Mock group"
-        class="Button-module_button__2ZuB7 i__imported_focusable_0 Button-module_small__l39oh Button-module_quiet__1KlhF Button-module_secondary__R0waJ"
+        class="Button-module_button__2ZuB7 utility-module_focusable__1fBKr Button-module_small__l39oh Button-module_quiet__1KlhF Button-module_secondary__R0waJ"
```

Comment:
I believe this was a downgrade - maybe you should merge from main and update the snapshot again? 🤔

Reply:
Thank you ❤️